slnpatch.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+script_dir = os.path.dirname(__file__)
+
+from buildtools import log
+from buildtools.buildsystem.visualstudio import VisualStudio2015Solution, ProjectType
```

Comment:
Just a note: `buildtools` is from https://gitlab.com/N3X15/python-build-tools, which is a hairy mess of utilities I've collected over the years.

Do the usual `python setup.py install` and it should work.